### PR TITLE
tested getters and createBoard methods with different size rows and c…

### PR DESCRIPTION
**Pull request by EVWatson**
…olumns

tests all passing

### CODE REVIEW

File: Tic_Tac_Toe_V2.1/test/BoardTest.java [hunk 1]
```diff
@@ -81,7 +81,21 @@ public void methodGetCurrentBoard_whenGivenALargerNumberOfRowsThanColumns_return
     }
 
     @Test
-    public void methodCreateBoard_whenGivenNumberOfRowsAndColumns_returnsArrayOfStringsWithCorrectNumnberOfRowsAndColumns(){
+    public void methodCreateBoard_whenGivenNumberOfRowsAndColumns_returnsArrayOfStringsWithCorrectNumberOfRowsAndColumns(){
 
+        Board currentBoard = new Board(6, 6);
```
Comment (jonathan-loe) [inline Tic_Tac_Toe_V2.1/test/BoardTest.java]: I understand this test is trying to test `createBoard()` method.

could you pls tell me what `Board currentBoard = new Board(6, 6)` is for? :)

Comment (EVWatson) [reply Tic_Tac_Toe_V2.1/test/BoardTest.java]: its for testing if the board is still ok if given dimensions other than 3x3

Comment (jonathan-loe) [reply Tic_Tac_Toe_V2.1/test/BoardTest.java]: sorry Emily - I can't see that in the test (or I don't quite follow you). could you pls elaborate that more? 😞

Comment (EVWatson) [reply Tic_Tac_Toe_V2.1/test/BoardTest.java]: hmm i think maybe i don't understand what you're asking? might be easier to go over in person tomorrow

Comment (EVWatson) [reply Tic_Tac_Toe_V2.1/test/BoardTest.java]: oh hang on. you mean, what is Board currentBoard = new Board(6, 6). Its a new instantiation of object Board, with 6 rows and 6 columns given as arguments.

File: Tic_Tac_Toe_V2.1/src/Board.java [hunk 2]
```diff
@@ -12,18 +12,39 @@ public Board(int totalRows, int totalColumns) {
...
 
+//    made createBoard public in order to test it
+
     private String[][] createBoard(int rowSize, int columnSize){
         String[][] newBoard = new String[rowSize][columnSize];
 
         for (int row = 0; row < newBoard.length; row ++) {
             for (int column = 0; column < newBoard[row].length; column++) {
                 newBoard[row][column] = "*";
+
```
Comment (jonathan-loe) [inline Tic_Tac_Toe_V2.1/src/Board.java:23]: 👾

**Pull request merged**